Documentation/network/servicemesh/l7-traffic-management.rst

```diff
@@ -16,11 +16,30 @@ and CiliumClusterwideEnvoyConfig).
 Prerequisites
 #############
 
-* Cilium must be configured with ``kubeProxyReplacement`` as partial
-  or strict. Please refer to :ref:`kube-proxy replacement <kubeproxy-free>`
+* Cilium must be configured with ``kubeProxyReplacement`` as ``true``.
+  Please refer to :ref:`kube-proxy replacement <kubeproxy-free>`
   for more details.
 * The minimum supported Kubernetes version for Ingress is 1.19.
 
+Caveats
+#######
+
+* ``CiliumEnvoyConfig`` resources have only minimal validation performed, and
+  do not have a defined conflict resolution behavior. This means that if you
+  create multiple CECs that modify the same parts of Envoy's config, the results
+  may be unpredictable.
+* In addition to this minimal validation, ``CiliumEnvoyConfig`` has minimal
+  feedback to the user about the correctness of the configuration. So in the
+  event a CEC does produce an undesirable outcome, troubleshooting will require
+  inspecting the Envoy config and logs, rather than being able to look at the
+  ``CiliumEnvoyConfig`` in question.  
+* ``CiliumEnvoyConfig`` is used by Cilium's Ingress and Gateway API support to
+  direct traffic through the per-node Envoy proxies. If you create CECs that
+  conflict with or modify the autogenerated config, results may be unpredictable.
+  Be very careful using CECs for these use cases. (The above risks are managed
+  by ensuring that all config generated by Cilium is semantically valid, as far
+  as possible).
```

Comment:
Would it be possible to remove the parentheticals here?

Reply:
Good call, the parentheses were adding nothing, removed.